RFS/src/test/java/org/opensearch/migrations/bulkload/transformers/TransformFunctionsTest.java

```diff
@@ -0,0 +1,98 @@
+package org.opensearch.migrations.bulkload.transformers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.jupiter.api.Test;
+
+import lombok.extern.slf4j.Slf4j;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Slf4j
+public class TransformFunctionsTest {
+
+    @Test
+    public void removeIntermediateMappingsLevels_AsExpected() throws Exception {
+        // Extract from {"mappings":[{"_doc":{"properties":{"address":{"type":"text"}}}}])
+        ObjectNode testNode1 = new ObjectMapper().createObjectNode();
+        ArrayNode mappingsNode1 = new ObjectMapper().createArrayNode();
+        ObjectNode docNode1 = new ObjectMapper().createObjectNode();
+        ObjectNode propertiesNode1 = new ObjectMapper().createObjectNode();
+        ObjectNode addressNode1 = new ObjectMapper().createObjectNode();
+        addressNode1.put("type", "text");
+        propertiesNode1.set("address", addressNode1);
+        docNode1.set(TransformFunctions.PROPERTIES_KEY_STR, propertiesNode1);
+        ObjectNode intermediateNode1 = new ObjectMapper().createObjectNode();
+        intermediateNode1.set("_doc", docNode1);
+        mappingsNode1.add(intermediateNode1);
```

Comment:
could you replace this w/ inline json that gets parsed?
```
  var json = 
      "{" +
      "  \"mappings\": [{" +
      "    \"_doc\": {" +
      ...
```

It's a lot easier to read and to maintain.  You can copy-paste literal json from a text editor into your IDE and it should do all of the necessary escaping (IntelliJ does this, I'd hope eclipse & vscode do too).

Then you can just new `ObjectMapper().readTree(json)` and get the parsed JsonNode back.